src/quest_manager/templates/quest_manager/buttons.html

```diff
@@ -8,7 +8,15 @@
       </a>
     {% endif %}
 
-    {% if  request.user.is_staff or request.user == q.editor %}
+    {% if q.archived and request.user.is_staff %}
+      <form action="{% url 'quests:unarchive' q.id %}" method="post" style="display:inline">
+        {% csrf_token %}
+        <button type="submit" class="btn btn-default" title="Unarchive Quest - send it to the drafts tab">
```

Comment:
For consistency with all the other button title/help text which use colons instead of dashes:
"Unarchive this quest: move it into your drafts tab so you can edit it"